Alignment/OfflineValidation/README_DMR

```diff
@@ -0,0 +1,75 @@
+## DMR validation
```

Comment:
please add the file extension `.md` such that it gets formatted by gitHub.

Reply:
ok